package.json

```diff
@@ -22,7 +22,7 @@
     "events": "1.0.2"
   },
   "peerDependencies": {
-    "react-native": ">=0.35"
+    "react-native": ">=0.40”
```

Comment:
Should be fixed now.
Apologies for sloppiness- I'm fairly new to pull requests  :$